src/WorldStorage/WSSAnvil.cpp

```diff
@@ -3039,6 +3055,49 @@ void cWSSAnvil::LoadWolfOwner(cWolf & a_Wolf, const cParsedNBT & a_NBT, int a_Ta
 
 
 
+void cWSSAnvil::LoadOcelotOwner(cOcelot & a_Ocelot, const cParsedNBT & a_NBT, int a_TagIdx)
```

Comment:
Is there a cleaner/better way to do this?
https://github.com/cuberite/cuberite/pull/3829/files#diff-a4136529bdf04bfe56434fa0b68698a0R3027